action/protocol/execution/testdata-shanghai/self-destruct.json

```diff
@@ -61,10 +61,10 @@
         "rawGasLimit": 1000000,
         "rawGasPrice": "0",
         "rawAccessList": [],
-        "rawExpectedGasConsumed": 10400,
+        "rawExpectedGasConsumed": 12646,
         "expectedStatus": 1,
         "hasReturnValue": true,
-        "rawReturnValue": "",
+        "rawReturnValue": "0000000000000000000000000000000000000000000000000000000000000014",
```

Comment:
"selfdestruct" has been deprecated. Note that, starting from the Cancun hard fork, the underlying opcode no longer deletes the code and data associated with an account and only transfers its Ether to the beneficiary

Reply:
this is test for Shanghai EVM, your change should not break the test